.github/workflows/migration-engine.yml

```diff
@@ -82,14 +85,14 @@ jobs:
           TEST_DATABASE_URL: ${{ matrix.database.url }}
 
       - run: cargo test -p migration-engine-tests
-        if: ${{ !matrix.database.single_threaded }}
+        if: ${{ !matrix.database.is_cockroach && !matrix.database.single_threaded }}
         env:
           CLICOLOR_FORCE: 1
           TEST_DATABASE_URL: ${{ matrix.database.url }}
           RUST_LOG: debug
 
       - run: cargo test -p migration-engine-cli
-        if: ${{ !matrix.database.single_threaded }}
+        if: ${{ !matrix.database.is_cockroach && !matrix.database.single_threaded }}
```

Comment:
The CLI has few tests and they exercise things like `create_database`, maybe it's already working? (Please ignore this if you already tried)